lib/index.ts

```diff
@@ -196,12 +216,24 @@ function createDelayLoadingFile(meshOrGeometry: any, outputDir: string, filename
         result.subMeshes = meshOrGeometry.subMeshes;
     }
 
-    const json = JSON.stringify(result, null, 0);
+    return JSON.stringify(result, null, 0);
+}
+
+function createDelayLoadingFile(
+    meshOrGeometry: any,
+    outputDir: string,
+    filename: string,
+    meshString: string,
+    mesh = true,
+) {
+    const encodedName = (mesh ? meshOrGeometry.name : meshOrGeometry.id).replace("+", "_").replace(" ", "_");
+    const outputPath = `${filename}.${encodedName}${mesh ? ".babylonmeshdata" : ".babylongeometrydata"}`;
+    const json = meshString ? meshString : createDelayLoadingString(meshOrGeometry, mesh);
```

Comment:
Not sure if I am missing something, but it doesn't look like `meshString` could be missing here. Probably it can directly replace the variable `json`.

Reply:
You are right, fixed